reference/functions/execresult.markdown

```diff
@@ -43,7 +43,8 @@ only.  Effectively calls to this function will be also repeatedly
 executed by `cf-promises` when it does syntax checking, which is
 highly undesirable if the command is expensive.  Consider using
 `commands` promises instead, which have locking and are not evaluated
-by `cf-promises`.
+by `cf-promises`. If capturing stderr is undesirable, consider **useshell** and
+redirecting it to `/dev/null`.
```

Comment:
Do you want to give an example of this?

Reply:
OK, I adjusted the existing example in core.

https://github.com/cfengine/core/pull/3759